File: JavaScript/1005.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var x = (a*3.5+b*7.5)/(3.5+7.5)

console.log("MEDIA = " + x.toFixed(5))

File: C/1005.c
#include <stdio.h>
 
int main() { 
    double a, b, x;
    
    scanf("%lf", &a);
    scanf("%lf", &b);

    x = (a*3.5+b*7.5)/(3.5+7.5);

    printf("MEDIA = %.5lf\n", x);

    return 0;
}